gwpy/timeseries/io/nds2.py

```diff
@@ -34,6 +34,7 @@
 from ...utils import gprint
 from ...utils.progress import progress_bar
 from .. import (TimeSeries)
+from .core import _pad_series
```

Comment:
@duncanmmacleod, if the `_pad_series` utility is now being imported from another module, should it no longer be private?

Reply:
IMO, no, but only because it's being imported by another module in the same package. The underscore denotes that this is private in the scope of the package, not the module.

Reply:
Ah, for some reason I'd always thought that was rule was at the module level, but this makes sense, thank you.